files/en-us/web/api/web_share_api/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: Web Share API
+slug: Web/API/Web_Share_API
+tags:
+  - API
+  - Apps
+  - Web Share
+  - Web Share API
+  - Guide
+  - Overview
+---
+<div>{{DefaultAPISidebar("Web Share API")}}</div>
+
+<p>The <strong>Web Share API</strong> provides a mechanism for sharing text, links, files and other content to an arbitrary <em>share target</em> selected by the user.</p>
+
+<p>The available share targets are provided using the native mechanisms of the underlying operating system. They might, for example, include the system clipboard or other services, Bluetooth or WiFi, email, contacts or messaging applications, and websites.</p>
+
+<p>The API consists of just two methods:</p>
+
+<ul>
+  <li>{{domxref("navigator.canShare()")}}: Validate that the specified data is sharable (check if a call with the same data to {{domxref("navigator.share()")}} would succeed).</li>
```

Comment:
```suggestion
  <li>{{domxref("navigator.canShare()")}}: Returns a boolean indicating whether the specified data sharable; indicates whether a call with the same data to {{domxref("navigator.share()")}} would succeed.</li>
```

Reply:
@jpmedley I'm proposing and going to accept thismodification of your wording to remove the text after the semicolon. The method does check if the data is sharable, but there are other reasons why `navigator.share()` might fail.
```suggestion
  <li>{{domxref("navigator.canShare()")}}: Returns a boolean indicating whether the specified data sharable.</li>
```

We could keep the sentence and clarify that, but I think the first part captures the concept sufficiently.